index.html

```diff
@@ -20,8 +21,12 @@ <h1 class="header__title">Découverte de l'Open Source</h1>
 				</p>
 
 				<div class="button__bar">
-					<button class="header__button">Comment participer ?</button>
-					<button class="header__button">GitHub</button>
+					<a href="#comment_participer">
+						<button class="header__button">Comment participer ?</button>
+					</a>
+					<a href="https://github.com/Eirbware/decouverte-open-source" target="_blank">
+						<button class="header__button">GitHub</button>
+					</a>
```

Comment:
`a` elements must have no interactive content descendant (https://www.w3.org/TR/2011/WD-html5-20110405/text-level-semantics.html#the-a-element)

```suggestion
					<button class="header__button" onclick="location.hash = '#comment_participer'">Comment participer ?</button>
					<button class="header__button" onclick="window.open('https://github.com/Eirbware/decouverte-open-source', '_blank').focus();">GitHub</button>
```